who_l3_smart_tools/utils/jinja2.py

```diff
@@ -0,0 +1,32 @@
+import os
+
+from jinja2 import Environment, FileSystemLoader
+
+DATA_TYPE_MAP = {
+    "Boolean": "boolean",
+    "String": "string",
+    "Date": "date",
+    "DateTime": "dateTime",
+    "Coding": "choice",
+    "ID": "string",
+    "Quantity": "integer",
+}
+
+
+def initalize_jinja_env(file_name):
+    template_dir = os.path.join(
+        os.path.dirname(os.path.abspath(file_name)), "templates"
+    )
+    return Environment(
+        loader=FileSystemLoader(template_dir),
```

Comment:
Have you considered using the PackageLoader for this? I found it pretty simple to use, e.g.,

```python
def initalize_jinja_env(module_name):
    return Environment(
        loader=PackageLoader(module_name), # defaults to using "templates" folder inside the package
        trim_blocks=True,
        lstrip_blocks=True,
    )
```

Then you replace:

```python
jinja2_env = initalize_jinja_env(__file__)
```

With:

```python
jinja2_env = initalize_jinja_env(__name__)
```